src/CloudWatchLogs/CloudWatchLogsClient.php

```diff
@@ -155,4 +156,46 @@
  * @method \Aws\Result updateLogAnomalyDetector(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateLogAnomalyDetectorAsync(array $args = [])
  */
-class CloudWatchLogsClient extends AwsClient {}
+class CloudWatchLogsClient extends AwsClient {
+    static $streamingCommands = [
+        'StartLiveTail' => true
+    ];
+
+    public function __construct(array $args)
+    {
+        parent::__construct($args);
+        $this->appendClientCustomMiddlewares();
+    }
+
+    private function appendClientCustomMiddlewares()
+    {
+        $this->getHandlerList()->appendInit($this->getSetStreamingFlagMiddleware(), 'streaming-flag-middleware');
+    }
+
+    private function getSetStreamingFlagMiddleware(): callable
```

Comment:
the "getSet" part of the name could be a little confusing.  We could name this `getStreamingFlagMiddleware` or something similar